media-video/ffmpeg/ffmpeg-4.0.1.recipe

```diff
@@ -156,9 +185,14 @@ PATCH()
 	sed -i "s,/usr/bin/perl,$portPackageLinksDir/cmd~perl/bin/perl," \
 		doc/Doxyfile \
 		doc/texi2pod.pl
+	# force no-undefined-symbols
+	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+		sed -i "s/SHFLAGS='-shared/SHFLAGS='-shared -Wl,--no-undefined/g" \
```

Comment:
Why is this needed? And wy only on gcc2?

Reply:
Because this will then error at build time instead of runtime if GCC uses some libgcc function we don't have.